src/gui/userstatus.cpp

```diff
@@ -66,19 +86,15 @@ void UserStatus::slotFetchUserStatusFinished(const QJsonDocument &json, const in
         qCInfo(lcUserStatus) << "Slot fetch UserStatus finished with status code" << statusCode;
         qCInfo(lcUserStatus) << "Using then default values as if user has not set any status" << defaultValues;
     }
+    
     const auto retrievedData = json.object().value("ocs").toObject().value("data").toObject(defaultValues);
     const auto emoji = retrievedData.value("icon").toString();
     const auto message = retrievedData.value("message").toString();
     auto statusString = retrievedData.value("status").toString(); 
-    _status = stringToEnum(statusString);
-    
-    // to display it to the user like 'Invisible' instead of 'invisible'
-    statusString.replace(0, 1, statusString.at(0).toUpper());  
 
-    const auto visibleStatusText = message.isEmpty()
-                                ? _status == DoNotDisturb? tr("Do not disturb") 
-                                                : tr(qPrintable(statusString))
-                                : message;
+    _status = stringToEnum(statusString);
+    statusString = enumToString(_status);
```

Comment:
@camilasan IMHO, this line is redundant. You can just write:
`const auto visibleStatusText = message.isEmpty() ? enumToString(_status) : message;` after all. It made me think more when reading this, because, I see this `statusString` is assigned with value before, and then, it is assigned with a new value right after it :)

You may shorter it even more if you just write:
```
_status = stringToEnum(retrievedData.value("status").toString());
const auto visibleStatusText = message.isEmpty() ? enumToString(_status) : message;
```

So, this intermediate variable `statusString` is not needed really.